client/src/routes/app.ts

```diff
@@ -0,0 +1,39 @@
+import Login from '../pages/Login/Login';
+import Signup from '../pages/SignUp/SignUp';
+import Dashboard from '../pages/Dashboard/Dashboard';
+import Settings from '../pages/Settings/Settings';
+import { AccountType } from '../types/AccountType';
+
+export interface appRouteType {
+  to: string;
+  component: React.ComponentType;
+  canView: Array<string>;
+  exact: boolean;
+}
+
+export const appRoutes = [
```

Comment:
nit: `export const appRoutes: AppRouteType[] = [ . . . ]`

Reply:
updated this too - thanks!